manuscript/build-container-image/kbld.md

```diff
@@ -21,12 +21,15 @@ Fun fact: `kbld` can work with multiple changing applications simultaneously, wh
 
 ```bash
 # note that in the deployment definition the container image is 'cncf-demo',
-#  and the kbld configuration points image 'cncf-demo' to our current directory
+#  and the kbld configuration points image 'cncf-demo' to our current directory, which contains our source code
```

Comment:
If we're writing longer comments, it might make sense to format them correctly instead of using comments. Something like:

------------------------------
Note that in the deployment definition the container image is 'cncf-demo', and the kbld configuration points image 'cncf-demo' to our current directory, which contains our source code

```bash
cat kbld/deployment.yaml
```

Build an image from the referenced source code and record the resulting image ref in a new file called `deployment-kbld.yaml`

```bash
kbld --file kbld/deployment.yaml | tee kbld/deployment-kbld.yaml
```
------------------------------

That will (probably) look much better in markdown preview as well as if we output it to other formats (e.g., book).

Reply:
(´ ▽｀).。ｏ♡ 
I love this and will implement it moving forward.